nimble/controller/src/ble_ll_adv.c

```diff
@@ -3613,6 +3613,56 @@ ble_ll_adv_ext_set_param(const uint8_t *cmdbuf, uint8_t len,
     return rc;
 }
 
+#if MYNEWT_VAL(BLE_VERSION) >= 54
+static uint8_t
+ble_ll_adv_ext_set_coded_opt(uint8_t phy_opt)
```

Comment:
this is not a setter function so 'set' in name is confusing

e.g. `ble_ll_adv_ext_coded_phy_mode_get`

also you can make func that takes both phy and phy_opt as parameter and call it `ble_ll_adv_ext_phy_mode_get`, this way you don't need extra checks for coded by later in the code - just always call this one